test/specs/lwcLsp.e2e.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import { TextEditor } from 'wdio-vscode-service';
+import { TestSetup } from '../testSetup';
+import * as utilities from '../utilities';
+
+describe('LWC LSP', async () => {
+  let testSetup: TestSetup;
+  let projectName: string;
+
+  step('Set up the testing environment', async () => {
+    testSetup = new TestSetup('LwcLsp', false);
+    await testSetup.setUp();
+    projectName = testSetup.tempProjectName.toUpperCase();
+
+    // Create Lightning Web Component
+    await utilities.createLWC('lwc1');
+  });
+
+  step('Verify Extension is Running', async () => {
+    // Using the Command palette, run Developer: Show Running Extensions
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Developer: Show Running Extensions',
+      1
+    );
+    let buttons = await $$('a.monaco-button.monaco-text-button');
+    for (const item of buttons) {
+      const text = await item.getText();
+      if (text.includes('Install and Reload')) {
+        await item.click();
+      }
+    }
+    await utilities.pause(10);
+    buttons = await $$('a.monaco-button.monaco-text-button');
+    for (const item of buttons) {
+      const text = await item.getText();
+      if (text.includes('Reload and Enable Extensions')) {
+        await item.click();
+      }
+    }
+    await utilities.pause(5);
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Developer: Reload Window',
+      10
+    );
+
+    // Verify Lightning Web Components extension is present and running
+    const extensionNameDivs = await $$('div.name');
```

Comment:
Also can you move this out to a utility function? Seems like a useful bit of code, and I see you're using almost the same in the aura lsp

Reply:
@CristiCanizales did you have any issues moving this one out?